src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -137,3 +137,14 @@ If you're encountering errors, double-check the version of `node` you're using l
 Cloudflare requires [Node v16.13](https://miniflare.dev/get-started/cli#installation), which is a more recent version than Astro’s out-of-the-box minimum, so double check you’re using at least v16.13.
 
 Client-side hydration may fail as a result of Cloudflare's Auto Minify setting. If you see `Hydration completed but contains mismatches` in the console, make sure to disable Auto Minify under Cloudflare settings.
+
+If you are deploying or building a project that is using server-side rendering and the server fails to build with error messages similar to:
+
+`[Error] Could not resolve "XXXX` `The package "XXXX" wasn't found on the file system but is built into node. Are you trying to
+  bundle for node? You can use "platform: 'node'" to do that, which will remove this error.`
```

Comment:
I wonder if it is helpful to add an actual example using a package, e.g. `fs`

Reply:
I didn't specify a particular package because I didn't future developers to associate the error message with the usage of a specific package. My idea was if we kept it general then it should be explicit that this can occur with any supported Node.js runtime package but if this is confusing I wouldn't be opposed to changing it to use an actual package.

Reply:
Understood. Let's wait what docs thinks about this :)